metrics/prometheus/prometheus.go

```diff
@@ -440,6 +441,11 @@ func NewMetrics(cfg config.PrometheusMetrics, disabledMetrics config.DisabledMet
 		[]string{adapterLabel},
 		standardTimeBuckets)
 
+	metrics.bidderServerResponseTimer = newHistogram(cfg, reg,
+		"bidder_server_response_time",
+		"Seconds to make a roundtrip to an adapter's bid server",
+		standardTimeBuckets)
```

Comment:
should  [add suffix describing the unit](https://prometheus.io/docs/practices/naming/) of `bidderServerResponseTimer`

```
bidder_server_response_time -> bidder_server_response_time_seconds
```

Also how about add more context to description
```
Duration needed to send HTTP request and receive response back from bidder server.
```

Reply:
addressed in [640d3ae](https://github.com/prebid/prebid-server/pull/2722/commits/640d3ae675addee4ad9ce64a5f4b6709df930531)